nats.go

```diff
@@ -2509,6 +2510,39 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// caller must lock
+func (nc *Conn) getServers(implicitOnly bool) []string {
+	poolSize := len(nc.srvPool)
+	var servers = make([]string, 0)
+	for i := 0; i < poolSize; i++ {
+		if implicitOnly && !nc.srvPool[i].isImplicit {
+			continue
+		}
+		url := nc.srvPool[i].url
+		servers = append(servers, fmt.Sprintf("%s://%s", url.Scheme, url.Host))
+	}
+	return servers
+}
+
+// Servers returns the list of known server urls, including additional
+// servers discovered after a connection has been established.  If
+// authentication is enabled, use UserInfo or Token when connecting with
+// these urls.
+func (nc *Conn) Servers() []string {
+	nc.mu.Lock()
+	defer nc.mu.Unlock()
+	return nc.getServers(false)
+}
+
+// ImplicitServers returns only the server urls that have been discovered
```

Comment:
here is a patch passing tests in case we want to do the rename :) https://github.com/wallyqs/nats/commit/769ac302fd292fb7b47b6e4dc762d5f11e56bb9b